src/System.IO.Compression/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -72,7 +72,7 @@ public enum CompressionLevel : int
         {
             NoCompression = 0,
             BestSpeed = 1,
-            BestCompression = 9,    // Refer to "How to choose a compression level" above.
+            BestCompression = 6,    // Refer to "How to choose a compression level" above.
```

Comment:
Instead of hard-coding our "Optimal" value to be "6" in zlib, I think our "Optimal" value should map to "-1"/"Z_DEFAULT_COMPRESSION" in zlib.  See my comments on https://github.com/dotnet/corefx/issues/4475 for reasoning.